hooks/post/30-upgrade.rb

```diff
@@ -31,13 +31,15 @@ def import_subscriptions
 end
 
 def elasticsearch_message
-  return true unless Kafo::Helpers.execute('rpm -q elasticsearch')
-
+  if `rpm -q elasticsearch`.gsub!(/\A"|"\Z/, '') != nil
```

Comment:
The `/\A"|"\Z/` regex seems to match any line that begins or ends with a `"`, so not sure what rpm output should match that